Content.Shared/Sound/SharedEmitSoundSystem.cs

```diff
@@ -102,6 +103,14 @@ private void OnEmitSoundOnDrop(EntityUid uid, EmitSoundOnDropComponent component
         TryEmitSound(uid, component, args.User);
     }
 
+    private void OnEmitSoundOnInteractUsing(EntityUid uid, EmitSoundOnInteractUsingComponent component, InteractUsingEvent args)
```

Comment:
new code should do `(Entity<EmitSoundOnInteractUsingComponent> ent, ref InteractUsingEvent args)`

Reply:
Could you explain a tad further? Why should this function definition be different than all the other handlers?

Reply:
Please disregard, I just had a dive into how the event bus works and that function declaration is an option.